cas/cas.go

```diff
@@ -59,9 +61,10 @@ var diskvStores = [...]string{
 
 // Store encapsulates a content-addressable-storage for storing ACIs on disk.
 type Store struct {
-	base   string
-	stores []*diskv.Diskv
-	db     *DB
+	base      string
+	stores    []*diskv.Diskv
+	db        *DB
+	treestore *TreeStore
```

Comment:
Correct. As my intention was to keep the treestore functions low level and adding exported functions inside the cas.Store (like done with Store.RenderTreeStore) I added other three functions:

* CheckTreeStore(key string) error
* GetTreeStoreRootFS(key string) string
* GetTreeStorePath(key string) string

I'm not sure it both GetTreeStorePath and GetTreeStoreRootFS are needed or the latter can be computed joining "rootfs" to the GetTreeStorePath output.